### PR TITLE
Remove fkprofile (it is unused)

### LINKED ISSUE #120: ./manage.py dumpdata fail with 'CommandError: Unable to serialize database: no such table: fkprofile_profile'
When trying to run the dumpdata command, it starts dumping json content and then stop with this error:
```
CommandError: Unable to serialize database: no such table: fkprofile_profile
```

I have no idea what is wrong.

### PR DESCRIPTION
**Pull request by odinho**
Some of the migration changes are actually python3-related.

I originally added migration for `fkprofile`, but that's stupid as I always wanted to kill it anyway. So here's that.

Fixes #120

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
Kan det være en ide å lage flere innsjekkinger (eller kanskje til og med flere endringsforlsag), en for python3-relaterte endringer og en for fjerningen av fkprofile?  Det er litt uklart for meg hvorfor url-strukturen endres samtidig med fjerningen.  Kan du forklare litt nærmere hvordan dette henger sammen?

**Comment by odinho** [comment]
Gjort. Eg var berre lat. Skulle berre fiksa den fkprofile-feilen din.

**Comment by odinho** [comment]
URL-ane endrast fordi eg flytta organization_list som låg i `fkprofile`, til `fk`, -- og la han i `/organization/` i staden, fordi det gav meining.

Då vil sjølvsagt lenker brekka viss nokon sjekker `/member/` i dag.

File: fkbeta/fk/views.py [hunk 1]
```diff
@@ -31,3 +36,14 @@ def user_profile(request):
         profile_form = UserProfileForm(instance=request.user)
         user_form = UserForm(instance=request.user)
     return render(request, "fk/profile.html", {'user_form': user_form, 'profile_form': profile_form})
+
+
+def member_organizations_list(request):
```
Comment (petterreinholdtsen) [inline fkbeta/fk/views.py]: This part of the patch is a bit confusing, as it is supposed to be moved from fkprofile, but it is only added, the the code in fkprofile is removed in the next commit.  Just mentioning it here to let everyone know what is happening.

**Comment by petterreinholdtsen** [comment]
One thing occured to me.  Why is there no migration script in the pull request to remove the profile and adjust the help text?  I tested the change on frikanalen-dev, and it seem to not break the web pages.

**Comment by odinho** [comment]
It'll just leave the empty table there. No problem with that.

**Pull request merged**